<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUserDirectories</class>
 <widget class="QDialog" name="ManageUserDirectories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage User Directories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDataSearch">
      <attribute name="title">
       <string>Data Search Directories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gbDataSearch">
         <property name="title">
          <string>Data Search Directories</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="leDirectoryPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbDataAddDirectory">
              <property name="text">
               <string>Add Directory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QListWidget" name="lwDataSearchDirs"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pbDataBrowseToDir">
                <property name="text">
                 <string>Browse To Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDataRemDir">
                <property name="text">
                 <string>Remove Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDataMoveToTop">
                <property name="text">
                 <string>Move to Top</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDataMoveUp">
                <property name="text">
                 <string>Move Up</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDataMoveDown">
                <property name="text">
                 <string>Move Down</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lblSearchArchive">
               <property name="text">
                <string>Search Data Archive:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbSearchArchive">
               <property name="maximumSize">
                <size>
                 <width>204</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Set custom value in Mantid.user.properties</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbDefaultSave">
         <property name="title">
          <string>Default Save Directory</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="leDefaultSave"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbSaveBrowse">
            <property name="text">
             <string>Browse to Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPythonDirectories">
      <attribute name="title">
       <string>Python Script Directories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="leDirectoryPathPython"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbScriptAddDirectory">
           <property name="text">
            <string>Add Directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="lwScriptSearchDirs"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pbScriptBrowseToDir">
             <property name="text">
              <string>Browse To Directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbScriptRemDir">
             <property name="text">
              <string>Remove Directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbScriptMoveUp">
             <property name="text">
              <string>Move Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbScriptMoveDown">
             <property name="text">
              <string>Move Down</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbPythonExt">
         <property name="title">
          <string>Python Extensions (fit functions, algorithms)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QListWidget" name="lwExtSearchDirs"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="pbExtBrowseToDir">
              <property name="text">
               <string>Browse to Directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbExtRemoveDir">
              <property name="text">
               <string>Remove Directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbExtMoveUp">
              <property name="text">
               <string>Move Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbExtMoveDown">
              <property name="text">
               <string>Move Down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layButtons">
     <item>
      <widget class="QPushButton" name="pbHelp">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
